hal/src/nRF52840/rgbled_hal.c

```diff
@@ -126,7 +126,6 @@ void LED_Init(Led_TypeDef Led) {
         return;
     }
 
-    HAL_Pin_Mode(HAL_Leds[Led].pin, OUTPUT);
```

Comment:
Does this affect RGB LED pins in any way?

Reply:
We use PWM for RGB LED, PWM will overwrite GPIO configuration, so RGB LEDs work correctly.
But it will affect USER LED.

`LED_Init()` is only called once to initialize all LEDs at boot, if we don't configure it output, USER LED should not work.
```
HAL_Core_Led_Mirror_Pin() -> HAL_Led_Init() -> LED_Init()
Set_System() -> LED_Init()
```
Gen2 should have this issue too, I found Gen2 also configures USER LED as output in `LED_Init()`, should we leave the output configuration?

Reply:
:+1:

I see no reason why we should by default configure it as output. We should double check the current behavior for Gen 2 though.